File: deeplynx.UI/src/app/(home)/types/responseDTOs/permissionResponseDto.ts
export type GroupResponseDto = {
  id: number|string;
  name: string;
  description?: string | null;
  action:string;
  resource?:string|null;
  isHardcoded:boolean;
  labelId?:number|string;
  lastUpdatedAt?: Date;
  lastUpdatedBy?: string | null;
  isArchived:boolean;
  projectId?: number | string;
  organizationId?: number | string;
}

File: deeplynx.models/ResponseDTOs/PermissionResponseDto.cs
namespace deeplynx.models;

public class PermissionResponseDto
{
    public long Id { get; set; }
    public string Name { get; set; }
    public string? Description { get; set; }
    public string Action { get; set; }
    public string? Resource { get; set; }
    public bool IsDefault { get; set; }
    public long? LabelId { get; set; }
    public DateTime LastUpdatedAt { get; set; }
    public string? LastUpdatedBy { get; set; }
    public bool IsArchived { get; set; }
    public long? ProjectId { get; set; }
    public long? OrganizationId { get; set; }
}